workflows/prognostic_c48_run/tests/test_emulator.py

```diff
@@ -1,39 +0,0 @@
-import tensorflow as tf
-import joblib
-from runtime.transformers.emulator import Config, Adapter
-from runtime.transformers.core import StepTransformer
-from fv3fit.emulation.thermobasis.emulator import Config as MLConfig
-
-
-def regression_state(state, regtest):
-    for v in sorted(state):
-        print(v, joblib.hash(state[v].values), file=regtest)
-
-
-def test_state_regression(state, regtest):
-    regression_state(state, regtest)
-
-
-def test_adapter_regression(state, regtest):
```

Comment:
@oliverwm1 I thought you touched this test recently. Is your PR merged yet?

Reply:
nvm. it looks like it was on a look-alike test: cb75ce82025645830e0051ed1e7e30f853b88467

Reply:
Yes, that was merged.